### PR TITLE
New abno: Fairy Gentleman

### PR DESCRIPTION
**Pull request by EnderMage99**
Fairy Gentleman

## About The Pull Request

Class: TETH
Counter 2
Work Damage: Medium Red

Work Types
Instinct: Very High
Insight: Common
Attachment: High
Repression: Low

-  After getting a good result with the Fairy Gentleman, He gives the employee a bottle of what he calls “Fairy Wine.”

-  Drinking the “Fairy Wine” will increase your prudence by 15 points and heal your sanity. However, Drinking this wine also will make the user drunk incredibly quickly. The positive effects only happen while the user is drunk. Employees who drank too much fairy wine felt better after taking x using a sleeper or using the pens you can buy in cargo.

- Doing instinct work with the Fairy Gentleman will also cause the agent to take a drink with the Gentleman.

## Why It's Good For The Game

More abno's are always better!

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/teth/fairy_gentleman.dm [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	say(pick(give_drink))
+	return
+
+/mob/living/simple_animal/hostile/abnormality/fairy_gentleman/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time)
+	switch(work_type)
+		if(ABNORMALITY_WORK_INSTINCT)
+			user.reagents.add_reagent(/datum/reagent/consumable/ethanol/fairywine, 10)
+			visible_message("<span class='notice'>You take a drink with the fairy gentleman. </span>")
+			say("Quite a lot of thanks for sharing this drink with me. Not many people often want to share this drink with me.")
+	return
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/teth/fairy_gentleman.dm:55]:
```suggestion
	if(work_type == ABNORMALITY_WORK_INSTINCT)
		user.reagents.add_reagent(/datum/reagent/consumable/ethanol/fairywine, 10)
		visible_message("<span class='notice'>You take a drink with the fairy gentleman. </span>")
		say("Quite a lot of thanks for sharing this drink with me. Not many people often want to share this drink with me.")
	return
```

File: code/modules/paperwork/records/info/teth.dm [hunk 2]
```diff
@@ -257,3 +257,12 @@
...
 	abno_breach_damage_type = "Pale"
 	abno_breach_damage_count = "Moderate"
+
+//Fairy Gentleman
+/obj/item/paper/fluff/info/teth/fairy_gentleman
+	abno_type = /mob/living/simple_animal/hostile/abnormality/fairy_gentleman
+	abno_code = "F-01-11-12"
+	abno_info = list(
+		"After getting a good result with the Fairy Gentleman, He gives the employee a bottle of what he calls “Fairy Wine.”",
+		"Drinking the “Fairy Wine” increase the prudence of the agent who drinks it and then heals their sanity. However, drinking this wine also will make the user become drunk incredibly quickly. Employees who drank too much fairy wine felt better after taking a special chemical using a sleeper or using the pens you can buy in cargo.",
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/teth.dm:267]: I don't think we have Penetic acid in pens in cargo, so it'd just be sleepers for this.

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/abnormality/teth/fairy_gentleman.dm (hunk 1 above)
Comment (EnderMage99) [reply code/modules/mob/living/simple_animal/abnormality/teth/fairy_gentleman.dm]: Yep, I think I have applied this change now.

File: code/modules/paperwork/records/info/teth.dm [hunk 3]
```diff
@@ -257,3 +257,12 @@
 		"When the abnormality found a corpse while breaching, it blessed a small area and destroyed the corpse. In this area, every employee recieved high PALE damage and every abnormality's Qliphoth Counter was lowered by 1.")
 	abno_breach_damage_type = "Pale"
 	abno_breach_damage_count = "Moderate"
+
+//Fairy Gentleman
+/obj/item/paper/fluff/info/teth/fairy_gentleman
+	abno_type = /mob/living/simple_animal/hostile/abnormality/fairy_gentleman
+	abno_code = "F-01-11-12"
```
Comment (Coxswain-Navigator) [inline code/modules/paperwork/records/info/teth.dm:264]: Change this, talk to me about getting it an actual number. We are not using limbus numbers.

**Review by Kitsunemitsu — APPROVED** [review]
I meant to merge this like 10 days ago, merging.

**Pull request merged**